### DISCUSSION TITLE
tailwindcss Why cannot build tr:focus?

### DISCUSSION THREAD

**Discussion by gyull0210** [tailwindlabs/tailwindcss#11540] [Help]
I tried to apply a background color to the `tr` tag of a table when it's focused by using `focus:bg-blue-600`, but it didn't work. I checked the CSS file and found that the corresponding code was not present. So, I directly added the code to the built output file, and it worked correctly. However, when I tried adding the same code to the CSS file that is read during the build process, it still didn't get included in the build. What could be the reason for this?

**Comment by wongjn** [comment]
Consider checking:
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.

**Reply by gyull0210** [reply]
I found it. my mistake. `focus:bg-blue-600` is OK. but `focus:(bg-blue-600 text-white)` is not OK. thank you. wong.
